laborator/content/operatii-memorie-gdb/2-delete-first/delete-first.c

```diff
@@ -5,6 +5,10 @@
 char* delete_first(char *s, char *pattern);
 
 int main(){
+	/*
+	TODO: Este corecta declaratia variabilei s in contextul in care o sa apelam
```

Comment:
Use diacritics and mind the `checkpatch.pl` error below. Comments should look like this:

```c
/**
 * This is a comment.
 */
```